package.json

```diff
@@ -500,7 +500,8 @@
     "compile": "webpack --mode none",
     "watch": "webpack --mode none --watch",
     "pretest": "tsc -p ./",
-    "test": "node ./out/test/runTest.js"
+    "test": "node ./out/test/runTest.js",
+    "lint": "eslint src --ext ts"
```

Comment:
How is
```
 "lint": "eslint --config .eslintrc.js ./src/*.ts"
```
it will use `.eslintrc.js`

Reply:
eslint by default uses `.eslintrc.*` if exists. It's perfectly fine if we make it explicit.

Reply:
Great! I got it.